.changeset/tabaxi-fellas-inspire.md

```diff
@@ -0,0 +1,220 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+"__section": tree
+---
+TableSchema (alpha) APIs have been added
+
+A `TableSchema` utility has been added to Shared Tree for managing dynamic, tabular data.
+This new `TableSchema` namespace contains APIs for creating column, row, and table [node schema](https://fluidframework.com/docs/api/fluid-framework/treenodeschema-typealias).
+
+Note: these APIs require the use of [SchemaFactoryAlpha](https://fluidframework.com/docs/api/fluid-framework/schemafactoryalpha-class).
+
+> [!WARNING]
+> These APIs are in preview and are subject to change.
+> Until these APIs have stabilized, we do not recommend using them in production code.
+> We reserve the right to make breaking changes to these APIs, including their persisted data format.
```

Comment:
As recommended by our linter, "we" isn't super well defined in all the context this message may occur in. I suggest updating this and the comment on the namespace. Maybe something like:

```suggestion
> Until these APIs have stabilized, it is not recommended to use them in production code.
> They may be breaking changes to these APIs and their underlying data format.
```